data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -350,18 +284,16 @@ else if(nodeSelExpr.getChildCount()==2 || nodeSelExpr.getChildCount()==1 && node
 			Relation relation = new Relation(conNumeric, null);
 			relations.add(relation);
 		} catch (NumberFormatException e) {
-			System.out.println("Unrecognizable column type");
+			LOGGER.info("Unrecognizable column type");
 		}
 		}
 		else
 			col = colName = nodeSelExpr.getChild(0).getChild(0).toString();
 if(col != null && colName != null) {
-	System.out.println("Input column = " + col);
+	LOGGER.info("Input column = " + col);
 	columnList.addToList("input->" + col);
 	Column column = new Column(null, colName, null, table_alias, EntityType.INTYPE, null, true);
 	if (label != null) column.setLabel(label);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00122)